realm/realm-library/src/main/java/io/realm/RealmQuery.java

```diff
@@ -404,6 +404,192 @@ public boolean isValid() {
         return this;
     }
 
+    // In
+
+    /**
+     * In comparison. This allows you to test if objects match any value in an array of values.
+     *
+     * @param fieldName the field to compare.
+     * @param values values to compare with and it cannot be null or empty.
+     * @return the query object.
+     * @throws java.lang.IllegalArgumentException if values is empty or do not match class or field type.
+     */
+    public RealmQuery<E> in(String fieldName, String... values) {
```

Comment:
The first version I made was the array version. @zaki50 suggested variant arguments version, I tested variant arguments several times and I adopted vargs instead of arrays.

In vargs version, we can invoke `in` operators by any fashions.

```
in("foo", "bar", "baz") // <- OK
in("foo", new String[] { "bar", baz" }) // <- OK
```

Although `Case case` parameter should not be the final parameter in the vargs versions, I thought it could be tolerable.